content/docs/cucumber/step-definitions.md

```diff
@@ -49,7 +49,7 @@ package com.example;
 import io.cucumber.java8.En;
 
 public class StepDefinitions implements En {
-    public StepDefinitions() {
+    public void StepDefinitions() {
```

Comment:
This is a constructor method. It doesn't need a return type. 

https://docs.oracle.com/javase/tutorial/java/javaOO/constructors.html

Reply:
I did not see it... That is an explanation why it did not work...